lib/apiservers/engine/backends/container.go

```diff
@@ -205,8 +205,13 @@ func (c *ContainerBackend) ContainerExecCreate(name string, config *types.ExecCo
 
 	var eid string
 	operation := func() error {
-		if vc.TryLock(APITimeout) {
+		op.Infof("ContainerExecCreate trying to lock vc: %s", name)
+		if vc.TryLock(time.Second) {
+			op.Infof("ContainerExecCreate successfully locked vc: %s", name)
 			defer vc.Unlock()
+		} else {
+			op.Infof("ContainerExecCreate cannot lock vc, continuing ..: %s", name)
```

Comment:
I think we should also get a message from the retry package - if that's the case we don't need this one.
We may want to rephrase it to note what we expect, e.g.
`ContainerExecCreate cannot lock container. Returning lock timeout error to prompt retry.`